contracts/Aavegotchi/facets/ItemsFacet.sol

```diff
@@ -190,7 +190,7 @@ contract ItemsFacet is Modifiers {
         onlyUnlocked(_tokenId)
     {
         Aavegotchi storage aavegotchi = s.aavegotchis[_tokenId];
-        require(aavegotchi.status == LibAavegotchi.STATUS_AAVEGOTCHI, "LibAavegotchi: Only valid for AG");
+        // require(aavegotchi.status == LibAavegotchi.STATUS_AAVEGOTCHI, "LibAavegotchi: Only valid for AG");
```

Comment:
Hi @narayanprusty and @KevinKons 

I was looking into this and that's what I've found:

I need to call `aavegotchiGameFacet.claimAavegotchi` to change the status to "AG"
But on this call, I'm having another error (Portal must be "open")... Then I'm trying to call `vrfFacet.openPortals`, but no luck so far and I'm getting stuck on this.

@narayanprusty some advices? Super thanks!